### DISCUSSION TITLE
Can´t make a payment to a sponsor

### DISCUSSION THREAD

**Discussion by Cipri84** [community/community#137503] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I can't make a payment to a sponsor. I´ve tried several times.
Could it be because the sponsor asks for payment in Dollars but my card is Euros?

**Comment by Mrsanyi123** [comment]
Hi @Cipri84  

The currency difference could be the issue. Your bank should convert Euros to Dollars, but there might be fees or restrictions on foreign transactions. Check your card settings or contact your bank to ensure it's set up for payments in Dollars.

**Comment by amirallami-code** [comment, marked as answer]
Yes, that could be the reason why you're having trouble making a payment to a GitHub sponsor. Here are a few things to consider:

1. Currency Mismatch: If the sponsor is asking for payment in US Dollars (USD) and your payment card is denominated in Euros (EUR), there could be a currency conversion issue. Your bank or payment processor may apply an unfavorable exchange rate, resulting in the payment not going through.

2. Payment Processor Restrictions: GitHub uses Stripe as their payment processor. Stripe may have certain restrictions or requirements around the currencies and payment methods they accept, which could be causing the issue if there's a mismatch between the sponsor's desired currency and your payment card currency.

3. International Transactions: Depending on your location and the sponsor's location, the payment may be considered an international transaction, which can sometimes trigger additional security checks or fees that could prevent the payment from completing successfully.

To resolve this, you could try the following:

- Use a payment card that is denominated in the same currency as the sponsor's request (USD in this case).
- Check with your bank or payment provider to see if they can facilitate the currency conversion and international transaction without any issues.
- Contact the sponsor and ask if they can provide an alternative payment method that may be more compatible with your payment card and location, such as a local currency option or a different payment processor.

If you've tried these steps and are still having trouble, you may need to reach out to GitHub support for further assistance in resolving the payment issue with the sponsor.

If you could mark this answer that would be much appreciated, thanks!

**Reply by Cipri84** [reply]
How do I contact the sponsor?

**Reply by amirallami-code** [reply]
You can simply email them or use any way of connections that they provide.

**Reply by Cipri84** [reply]
I don´t see any way of connection provided...
